t/app/controller/report_inspect.t

```diff
@@ -6,8 +6,8 @@ use FixMyStreet::TestMech;
 
 my $mech = FixMyStreet::TestMech->new;
 
-my $brum = $mech->create_body_ok(2514, 'Birmingham City Council');
-my $oxon = $mech->create_body_ok(2237, 'Oxfordshire County Council');
+my $brum = $mech->create_body_ok(2514, 'Birmingham City Council', id => 2514);
+my $oxon = $mech->create_body_ok(2237, 'Oxfordshire County Council', id => 2237);
```

Comment:
I hope that we can get rid of the UK cobrand's need for body ID = area ID at some point, and doing that would certainly not persuade us to ;)